src/babel/tools/resolve-rc.js

```diff
@@ -24,7 +25,7 @@ export default function (loc, opts = {}) {
       var json;
 
       try {
-        json = jsons[content] ||= JSON.parse(content);
+        json = jsons[content] || (jsons[content] = JSON.parse(stripJsonComments(content)));
```

Comment:
To store it in the cache - it's the same operation as  `||=`,  but since the playground is gone, I figured I shouldn't use it.